eng/pipelines/test-integration-job.yml

```diff
@@ -29,6 +29,15 @@ steps:
     displayName: Set Feature Flags
     condition: and(succeeded(), ne(variables['FeatureFlag'], ''))
 
+  - powershell: New-ItemProperty
+      -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem"
+      -Name "LongPathsEnabled"
+      -Value "1"
+      -PropertyType DWORD
+      -Force
+    displayName: Enable Long Paths
+    condition: succeeded()
```

Comment:
nit: `condition: succeeded()` is not needed (I know we have it in other steps as well though), it should be [the default](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops):
![image](https://github.com/user-attachments/assets/a38408ad-a1f2-4cbc-943a-d4824b79a8e6)